my-webxr-app/template.env

```diff
@@ -0,0 +1,3 @@
+# Clone and rename to .env
+VITE_ROLLBAR_ACCESS_TOKEN=<ROLLBAR_API_KEY>
+VITE_ROLLBAR_ENVIRONMENT=local-dev
```

Comment:
@matthew-buglass We may want to get a secret on github set up for this ASAP if we dont have it done yet.

Reply:
I believe they've been added already 👍